Update README.md

giorgialodi opened the pull request:
Added a description of the types of harvesters in order to respond to issue #5

giorgialodi committed:
Update README.md

giorgialodi commented:
@jenkin could you please check whether the text is in line with what you required? Thanks!

Commit:
Fix typos and add links

jenkin commented:
@giorgialodi Some suggestions to replace problematic *resource* word:

* portal ("A **portal** is defined by two entities: an organization and a harvesting source")
* catalog ("A **catalog** is defined by two entities: an organization and a harvesting source")
* ...

giorgialodi commented:
@jenkin let's take the official defintiions:

- catalog: --> A data catalog is a curated collection of metadata about datasets. Curated means a process behind with organizations involved, harvesting sources are technical elements through which collecting metadata

- portal - according to Cambridge dictionary a portal is defined as a website that collects information on a particular subject and has links (= connections) to other websites with related information

Is the term "entitities"  not correct? Because it seams to me more appropriate and I proposed that change in the Readme. Do you need something to integrate both the two concepts (organizations - agents and harvesting sources - objects?)

Probably between the two I will used  data portal (and not open data so as to give the possible idea of a future where not only open data are categorizedwhich is the definition above managed by organizations.

And I would write "data portal: it embraces two entities; namely organizations and harvesting sources"

What do you think? Better?

Commit:
From resources to entities
- update README
- update validate script
- update export script
- change dist/ folder structure
